doc/sphinx/ref_output.rst

```diff
@@ -0,0 +1,64 @@
+.. role:: code-rst(code)
+   :language: reStructuredText
+
+Output Reference
+===============================
+After a run of the MDTF framework is completed, the output can be found in the 
```

Comment:
Processed data, ESM-intake catalogs, POD output, and logs from the MDTF-framework run are stored in a directory called MDTF_output that is appended to the OUTPUT_DIR (defaults to WORK_DIR if OUTPUT_DIR is not set) specified in the runtime configuration file. Each new run will generate an output directory with `_v#` appended to the `MDTF_output` base name if the OUTPUT_DIR contains directories from prior framework runs.